aerosonicdb/data/download.py

```diff
@@ -1,63 +1,96 @@
-from aerosonicdb.utils import get_project_root
-
-root_path = get_project_root()
-download_path = os.path.join(root_path, 'data/raw')
-
-
-def download(target_path=download_path):
-    # Target URLs for the dataset files on Zenodo
-    audio_zip = 'https://zenodo.org/record/10215080/files/audio.zip'
-    env_audio_zip = 'https://zenodo.org/record/10215080/files/env_audio.zip'
-    sample_meta_csv = 'https://zenodo.org/record/10215080/files/sample_meta.csv'
-    aircraft_meta_csv = 'https://zenodo.org/record/10215080/files/aircraft_meta.csv'
-    aircraft_meta_json = 'https://zenodo.org/record/10215080/files/aircraft_meta.json'
-    locations_json = 'https://zenodo.org/record/10215080/files/locations.json'
-    env_map_csv = 'https://zenodo.org/record/10215080/files/environment_class_mappings.csv'
-    raw_env_map = 'https://zenodo.org/record/10215080/files/environment_mappings_raw.csv'
-    license_txt = 'https://zenodo.org/record/10215080/files/LICENSE.txt'
-    readme_md = 'https://zenodo.org/record/10215080/files/README.md'
-
-    # Specify and create the target directory for the dataset
-
-    if not os.path.exists(target_path):
-        os.makedirs(target_path)
-
-        # Download the audio zip file
-        urllib.request.urlretrieve(audio_zip, os.path.join(target_path, 'audio.zip'))
-
-        # Download the environment audio zip file
-        urllib.request.urlretrieve(env_audio_zip, os.path.join(target_path, 'env_audio.zip'))
-
-        # Download the meta files
-        urllib.request.urlretrieve(sample_meta_csv, os.path.join(target_path, 'sample_meta.csv'))
-        urllib.request.urlretrieve(env_map_csv, os.path.join(target_path, 'environment_class_mappings.csv'))
-        urllib.request.urlretrieve(raw_env_map, os.path.join(target_path, 'environment_mappings_raw.csv'))
-        urllib.request.urlretrieve(aircraft_meta_csv, os.path.join(target_path, 'aircraft_meta.csv'))
-        urllib.request.urlretrieve(aircraft_meta_json, os.path.join(target_path, 'aircraft_meta.json'))
-        urllib.request.urlretrieve(locations_json, os.path.join(target_path, 'locations.json'))
-        urllib.request.urlretrieve(license_txt, os.path.join(target_path, 'LICENSE.txt'))
-        urllib.request.urlretrieve(readme_md, os.path.join(target_path, 'README.md'))
-
-        # Extract the dataset
-        with zipfile.ZipFile(os.path.join(target_path, 'audio.zip'), 'r') as zip_ref:
-            zip_ref.extractall(target_path)
-
-        # Clean up the zip file
-        os.remove(os.path.join(target_path, 'audio.zip'))
-
-        # Extract the environment dataset
-        with zipfile.ZipFile(os.path.join(target_path, 'env_audio.zip'), 'r') as zip_ref:
-            zip_ref.extractall(target_path)
-
-        # Clean up the zip file
-        os.remove(os.path.join(target_path, 'env_audio.zip'))
-
-    else:
-        print(f'Dataset downloaded - see "data/raw" directory')
-
-
-if __name__ == '__main__':
-    download()
+#!/usr/bin/env python
```

Comment:
adding a `#!/usr/bin/env python` is standard for python files you wish to make callable.
can then invoke them simply with `./download.py` (assuming they are runnable e.g. after running `chmod +x download.py`).

This is also a nice way to signal that the file includes a runnable entry point